runtime/services/catalog/migrator/sources/sources.go

```diff
@@ -317,6 +325,39 @@ func mergeFromParsedQuery(apiSource *runtimev1.Source) error {
 	return nil
 }
 
+func rewriteLocalRelativePath(ast *duckdbsql.AST, repoRoot string, allowRootAccess bool) (string, error) {
+	var resolveErr error
+	err := ast.RewriteTableRefs(func(table *duckdbsql.TableRef) (*duckdbsql.TableRef, bool) {
+		newPaths := make([]string, 0)
+		for _, p := range table.Paths {
+			if strings.Contains(p, "/") {
+				p, err := fileutil.ResolveLocalPath(p, repoRoot, allowRootAccess)
+				if err != nil {
+					resolveErr = err
+					return nil, false
+				}
+				newPaths = append(newPaths, p)
+			} else {
+				newPaths = append(newPaths, p)
+			}
```

Comment:
Why only when there's a `/` in the path? Wouldn't doing something like `select * from "xxx.csv"` load from process root instead of repo root in this case?